### ISSUE TITLE
[css-pseudo-4] Animations on ::marker?

### ISSUE THREAD

**Issue by andruud** [w3c/csswg-drafts#4814]
We have some WPTs (mostly tentative) that expect animations and transitions to work on ::marker ([see Chrome issue for list](https://bugs.chromium.org/p/chromium/issues/detail?id=1054509)).

However, it appears that the spec does not currently allow animation and transition properties on ::marker. I'm wondering if there has been been any discussion regarding allowing this, or if we think it's a given that animations and transitions will be supported at some point.

Basically I'm wondering if I should remove ::marker from these tests, or keep them (if animation support is inevitable). (Sidenote: The reason this came up in the first place, is that the tests suddenly change behavior if ::marker is supported (via `@supports selector`)).

cc @birtles

**Comment by MatsPalmgren** [comment]
> However, it appears that the spec does not currently allow animation and transition properties on ::marker.

What makes you think animation isn't allowed?

`::marker` is a [CSSPseudoElement](https://drafts.csswg.org/css-pseudo-4/#cssom) which makes it a target for animation (normative requirement). This was resolved in https://github.com/w3c/csswg-drafts/issues/3763#issuecomment-481773782.

**Comment by AmeliaBR** [comment]
> What makes you think animation isn't allowed?

There's an [explicit allow-list of properties](https://drafts.csswg.org/css-pseudo-4/#selectordef-marker) that currently apply to marker, until the full layout model can be specified.

My assumption would be that animation/transition of the allowed properties are also allowed & that the animation/transition properties apply — but I agree that this should be specified explicitly!

**Comment by Loirooriol** [comment]
Being able to animate the properties that apply to ::marker doesn't sound unreasonable. But note that ::first-letter and ::first-line also filter which properties apply to them, and you can't animate the allowed ones.

**Comment by fantasai** [comment]
I don't see a reason to disallow animations and transitions on ::marker, so adding them to the allow list makes sense.
